brainscore_language/plugin_management/conda_score.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+import pickle
+import tempfile
+
+from brainscore_core.metrics import Score
+from brainscore_language.plugin_management.environment_manager import EnvironmentManager
+
+TMP_DIR = tempfile.TemporaryDirectory()
```

Comment:
this creates a temporary directory upon import. I suggest to make `save_score` and `read_score` into methods of class `CondaScore` and initialize `TMP_DIR` on demand. This also avoids conflicts when `save_score` is called twice after one another before `read` so that the first `read` will get the second score.
```
def score_filepath(self) -> str:
    if self._scorepath is None:
        tmp_dir = tempfile.TemporaryDirectory()
        self._scorepath = f'{TMP_DIR.name}_score.pkl'
    return self._scorepath
```

Although I guess this has to be somewhat persisted across processes and `tempfile.TemporaryDirectory()` will always give you a new directory... If that's the case and it gets too complicated, scratch that and we can tackle it in the future if needed.